arangod/Agency/Job.cpp

```diff
@@ -613,6 +615,14 @@ void Job::addIncreasePlanVersion(Builder& trx) {
   }
 }
 
+void Job::addIncreaseCurrentVersion(Builder& trx) {
```

Comment:
Don't see, why you need to adjust Current/Version. Also we have tried to stay religiously on Plan side of things in Supervision / on coordinators and on `Current` on db servers